constants.js

```diff
@@ -43,7 +43,7 @@ module.exports = Object.freeze({
     TITLES_NEXT: 'Here are the next titles: ',
     TITLES_REPEAT: 'Repeating last titles<break />',
     TITLE_CHOOSE_SUMM_FULL:
-      'Would you like to hear a summary or the full article of ',
+      'Would you like to hear a summary or the full article?',
     TITLE_CHOICE_REPROMPT:
       'You can say summary to hear a summary or full article to hear the full article',
```

Comment:
I would change the reprompt to "You can say summary or full article"

Reply:
Changed!